supabase/supabase.service.ts

```diff
@@ -48,6 +48,29 @@ export class SupabaseService {
 
     return getUrl.data.publicUrl;
   }
+
+  async uploadPosterToBucket(file: Express.Multer.File): Promise<string> {
+    const fileName = generateUniqueFileName(file.originalname);
+    const { data, error } = await this.supabase.storage
```

Comment:
ensure that the bucket name for poster uploads is also configurable through the .env file